library/lcd/lcd_comm.py

```diff
@@ -116,32 +123,18 @@ def SendLine(self, line: bytes):
     def WriteLine(self, line: bytes):
         try:
             self.lcd_serial.write(line)
-        except serial.serialutil.SerialTimeoutException:
-            # We timed-out trying to write to our device, slow things down.
-            logger.warning("(Write line) Too fast! Slow down!")
         except serial.serialutil.SerialException:
-            # Error writing data to device: close and reopen serial port, try to write again
-            logger.error(
-                "SerialException: Failed to send serial data to device. Closing and reopening COM port before retrying once.")
-            self.closeSerial()
-            self.openSerial()
-            self.lcd_serial.write(line)
+            logger.warning("(WriteLine) error, reseting device.")
+            self.reset_by_usb()
```

Comment:
I think it's a shame to always reset USB and stop the program, when in some cases just the closeSerial/openSerial/write is enough to keep the program running (people have confirmed this in [#269](https://github.com/mathoudebine/turing-smart-screen-python/issues/269#issuecomment-1608869213))
Maybe the closeSerial/openSerial/write can be kept, and then only on exception the USB is reset?

Reply:
It's a good question... In case of desktop suspend/resume, only close and reopen the serial, don't work in 5 pol version.

Reply:
By the way, the official app on windows, send reset USB in case of errors in serial. It's try to open serial 2 times, if don't work, it's send the reset. I checked this behavior with Wireshark in USB.

Reply:
The 5 pol version its very strict to send images, updates, if the order is not following, it's doesn't work. 
For me, the correct is to stop all threads, reset USB and start again, but it's a massive change to the code, stop the program has less work to do, because the systemd cares about restart it.